spec/index.bs

```diff
@@ -1072,20 +1068,29 @@ GPUBuffer includes GPUObjectBase;
 
     : <dfn>\[[mapping]]</dfn> of type {{ArrayBuffer}} or {{Promise}} or `null`.
     ::
-        The mapping for this {{GPUBuffer}}.
+        The mapping for this {{GPUBuffer}}. The {{ArrayBuffer}} isn't directly accessible
+        and is instead accessed through view into it, the mapped ranges, that are stored
```

Comment:
1) reworded to "views into it, called the mapped ranges"...
2) Yes it is meant to be similar to JS users that know about typed array buffer views, because they also know you can create `ArrayBuffers` that are views into other `ArrayBuffers`. That's exactly what's happening.